ci/docker/docker_setup.sh

```diff
@@ -3,7 +3,9 @@ echo 'doing docker build'
 # Note: adding --build-arg <arg-name> without any value tells docker to
 #  use value from local environment (export DO_GIT_CLONE)
 
-docker build -t ${DOCKERHUB_TAG} --build-arg SOURCE_BRANCH=${DOCKERHUB_DEFAULT_TAGNAME} --build-arg MET_BRANCH=${DOCKERHUB_MET_TAGNAME} --build-arg DO_GIT_CLONE ${TRAVIS_BUILD_DIR}/internal_tests/docker
+#${TRAVIS_BUILD_DIR}/ci/docker/get_data_volumes.py
```

Comment:
I'm not entirely sure if these docker_* scripts belong in ci/docker. They are run by Travis. I think ci/docker should contain only things used exclusively by Docker, i.e. build hooks, Dockerfile, etc.